### PR TITLE
[IMP] mail, *: tests: remove fake mail.channel/members field

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2810539

### CODE REVIEW

File: addons/mail/static/tests/helpers/mock_server.js [hunk 1]
```diff
@@ -303,6 +303,21 @@ MockServer.include({
...
+        if (model === 'mail.channel') {
+            values2 = {
+                ...values,
+                channel_partner_ids: values.channel_partner_ids || [this.currentPartnerId],
+            };
+        } else {
+            values2 = values;
+        }
+        return this._super(model, values2);
+    },
```
Comment (seb-odoo) [inline addons/mail/static/tests/helpers/mock_server.js:320]: try to define the default on the field, that should work the same without this extra code

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

File: addons/im_livechat/static/tests/helpers/mock_server.js [hunk 2]
```diff
@@ -186,7 +186,7 @@ MockServer.include({
             // defaults that would set it to true) and here operator and visitor
             // can't be differentiated in that regard.
             'is_pinned': false,
-            'members': members, // channel_partner_ids
+            'channel_partner_ids': channel_partner_ids,
```
Comment (alexkuhn) [inline addons/im_livechat/static/tests/helpers/mock_server.js:189]: Omission on changing this, at first glance I thought it was `channelInfo` formatter but actually it's passed to `_mockCreate` so definitely needs conversion from `members` to `channel_partner_ids`,

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**